src/util/unicode.cpp

```diff
@@ -257,11 +257,13 @@ std::wstring utf8_to_utf16_native_endian(const std::string &in)
     return result;
 }
 
+/// Does not escape the '\'' character, as this is not required to be escaped in
```

Comment:
Say what it *does* do instead. Probably something like "escapes non-printable characters, whitespace except for spaces, double- and single-quotes and backslashes. This should yield a valid Java string literal"